app/models/content_length.rb

```diff
@@ -22,7 +22,7 @@ def self.destroy_all_and_recalculate
       next unless Rails.application.config.recalculate_content_length_if.call af
 
       CodecConversion.find_each do |cc|
-        af.delay(queue: :content_lengths_backlog).calc_audio_length(cc)
+        CalculateContentLengthJob.set(queue: :within_30_minutes).perform_later(af, cc)
```

Comment:
It's entirely infeasible to recalculate all content lengths (that qualify) within 30 minutes. This happens very rarely, but if it does it takes at least a few days. Is it possible to create a "no deadline" queue where these jobs can be scheduled to?

Reply:
Good point, I've renamed `:within_24_hours` to `:whenever` (as I don't think we need the distinction between those two, and the latter is more explicit).

Due to this change, I've put the `TranscodeCacheCleanJob` in `:within_30_minutes` as it should be possible for this job to be picked up rather quickly.